app/views/layouts/application.html.erb

```diff
@@ -24,6 +24,9 @@
   <%= csrf_meta_tags %>
 </head>
 <body>
+<!--[if ( !IE ) | ( gte IE 9 ) ]><!-->
+<%# render 'dough/shared/social_sharing_icon_sprite' %>
```

Comment:
If we are 100% certain that we will never ever use the SVG sprites in IE8 and below, then to me it makes sense to leave them un-rendered in that situation.